platforms/scripts/afu_platform_config

```diff
@@ -782,17 +784,17 @@ def findPlatforms(db_path):
     platforms = set()
     # Walk all the directories
     for db_dir in db_path:
-        try:
-            # Look for JSON files in each directory
-            for fn in os.listdir(db_dir):
-                if fn.endswith('.json'):
+        # Look for JSON files in each directory
+        for fn in os.listdir(db_dir):
+            if fn.endswith('.json'):
+                try:
                     with open(os.path.join(db_dir, fn)) as f:
                         # Does it have a platform name field?
                         db = json.load(f)
                         platforms.add(db['platform-name'])
-        except Exception:
-            # Give up on this file or directory if there is any error
-            None
+                except Exception:
+                    # Give up on this file or directory if there is any error
+                    None
```

Comment:
Do we want to give up on a directory if there is an error in just one JSON file? Wouldn't it be better to give up on just that file? Perhaps a little more explicit exception handling could help?